### PR TITLE
OSDOCS-1210: Release notes for removed API alerts

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-1210

Preview: https://deploy-preview-33491--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes#ocp-4-8-monitoring-removed-API-alerts

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@xingxingxia Can you please review?

Preview: https://deploy-preview-33491--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes#ocp-4-8-monitoring-removed-API-alerts

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -327,6 +327,16 @@ The Node Feature Discovery (NFD) Operator detects hardware features available on
 [id="ocp-4-8-monitoring"]
 === Monitoring
 
+[id="ocp-4-8-monitoring-removed-API-alerts"]
+==== Alerts and information on APIs in use that will be removed in the next release
+
+{product-title} 4.8 introduces two new alerts that fire when an API that will be removed in the next release is in use:
```
Comment (ahardin-rh) [inline release_notes/ocp-4-8-release-notes.adoc:333]: s/that fire/are triggered? Just thinking bout localization

**Comment by ahardin-rh** [comment]
Just one comment/suggestion. Looks great!

File: release_notes/ocp-4-8-release-notes.adoc (hunk 1 above)
Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc:333]: Good suggestion, though I do think that "fire" is used for alerts throughout the monitoring docs (at least the RN sections, which is where I checked for wording). I'll check w/ @lbarbeevargas to confirm!

Comment (bergerhoffer) [reply release_notes/ocp-4-8-release-notes.adoc:333]: Got :+1: from lindsey to use "alert"

**Comment by lbarbeevargas** [comment]
LGTM!
@simonpasquier FYI that this will also be included in the Monitoring release notes section.

**Pull request merged**